rocksdb_admin/admin_handler.cpp

```diff
@@ -605,7 +607,7 @@ void AdminHandler::async_tm_addS3SstFilesToDB(
     return;
   }
 
-  auto responses = s3Util->getObjects(request->s3_path, local_path);
+  auto responses = s3_util_->getObjects(request->s3_path, local_path);
```

Comment:
these block are all protected by the read lock at line 555, so if we are executing this line, there should be no opportunity for others the change the s3_util_.